cypress-tests/cypress/support/pageBuilder/pbDeleteAllCategories.ts

```diff
@@ -0,0 +1,81 @@
+import { GraphQLClient } from "graphql-request";
+
+declare global {
+    // eslint-disable-next-line @typescript-eslint/no-namespace
+    namespace Cypress {
+        interface Chainable {
+            pbDeleteCategoriesExceptStatic(): void;
+        }
+    }
+}
+
+const GET_CATEGORIES_QUERY = /* GraphQL */ `
+    query ListCategories {
+        pageBuilder {
+            listCategories {
+                data {
+                    slug
+                    name
+                    layout
+                    url
+                    createdOn
+                    createdBy {
+                        id
+                        displayName
+                    }
+                }
```

Comment:
I know we said we're gonna handle this in a separate PR.

I believe we can handle this immediately here.

So, in your code below, you just need `slug`, so let's just have that field included in the query.

Reply:
When I saw the change below, only using the slug made a lot more sense it is now the way you imagined it to be.